firmware/include/caseTempController/config.h

```diff
@@ -0,0 +1,12 @@
+//*****************************************//
+// configuration                           //
+//*****************************************//
+#define K_P				1	    // proportional factor of the PID controller
+#define K_I				0.1  	// integral factor of the PID controller
+#define K_D				0.5	    // derivative factor of the PID controller
+#define TAU				20	    // number of measurements viewed in calculation, reduce to save space and calculation times
+#define GOAL_TEMP		30	    // goal temperature of 30°C
+#define TEMP_GPIO		4	    // the GPIO data pin of the temperature sensor
+#define FAN_GPIO	    16      // the PWM pin for fan control
```

Comment:
pls check with with @felixherbst to match the korrekt pins

Reply:
pin 26 ist der temp sensor
pin 23 ist der utility fan

Reply:
Thank you, I'm changing it.

Reply:
Wouldn't it be better to have all pin assignments in one file instead of several?

Reply:
I can live with the way it currently is

Reply:
Maybe we can Chance this in the future though

Reply:
@mlkiefer You/We can open up an issue for this if you want to.